src/scenes/AcceleratorRouter/ParticipantProjects/Reports/ReportView.tsx

```diff
@@ -177,11 +177,11 @@ const ReportView = () => {
   );
 
   const year = selectedReport?.startDate.split('-')[0];
-  const quarterNumber = selectedReport?.startDate
-    ? Math.ceil((new Date(selectedReport?.startDate).getMonth() + 1) / 3)
-    : 0;
-  const reportName =
-    selectedReport?.frequency === 'Annual' ? `${year}` : quarterNumber ? `${year}-Q${quarterNumber}` : '';
+  const reportName = selectedReport
+    ? selectedReport.frequency === 'Annual'
+      ? `${year}`
+      : `${year}-${selectedReport.quarter}`
+    : '';
```

Comment:
Is there ever a chance that a startDate will be in the year prior to the end date? e.g. if a Q1 report was started in December due to timezone diffs or something?

Reply:
Hmm, I think it shouldn't happen, but I'm not 100% sure